src/clj_sqs_extended/aws/sqs.clj

```diff
@@ -11,11 +11,14 @@
            [com.amazonaws.services.sqs.model
             CreateQueueRequest
             DeleteMessageRequest
+            MessageAttributeValue
             PurgeQueueRequest
             ReceiveMessageRequest
             SendMessageRequest]))
 
 
+(def ^:private clj-sqs-ext-format-attribute "SerdesFormat")
```

Comment:
I'd give it some identifiable info, e.g. `clj-sqs-extended/SerdesFormat`

Reply:
That was pretty much what I tried first until I learned that there are restrictions in place:
https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-message-metadata.html

Reply:
could you note the reference here too?

I'd say `clj-sqs-extended.serdes-format`

Reply:
Roger that.